docs/core/extensions/includes/logging-samples-browser.md

```diff
@@ -0,0 +1,9 @@
+---
+author: IEvangelist
+ms.author: dapine
+ms.date: 07/30/2021
+ms.topic: include
+---
```

Comment:
I'm wondering if these metadata does anything for includes?

Reply:
From our internal contributor guides (we should really make these public):

![image](https://user-images.githubusercontent.com/7679720/127860640-227273d5-2e56-4e4f-960f-0121efd8c61c.png)

Reply:
> (we should really make these public):

Yess fully agree with that.